tests/test_utils.py

```diff
@@ -16,3 +16,15 @@ def test_base64encode_bytes():
 
     assert isinstance(encoded_string, str)
     assert encoded_string == 'dGhpcyBpcyBieXRlcyB0byBlbmNvZGU=\n'
+
+
+def test_iter_notifiers_stop_iteration():
```

Comment:
Please do not pile unrelated changes into one pull request, but make several.
Let that be the request only about AniLibria. Py3 related requests are welcome as separate.

Reply:
Sorry for that, you are definitely right.